src/wallet/rpcwallet.cpp

```diff
@@ -494,6 +497,14 @@ UniValue sendtoaddress(const JSONRPCRequest& request)
         }
     }
 
+    if (!request.params[8].isNull()) {
```

Comment:
It doesn't make sense to add a new param for this. Instead, make `params[6]` be conf target *or* feerate...